models/conversation.model.js

```diff
@@ -0,0 +1,32 @@
+import mongoose from "mongoose";
+
+const conversationSchema = new mongoose.Schema(
+  {
+    conversationType: {
+      type: String,
+      enum: ["DM", "Channel"],
+      required: true,
+    },
+    messages: [
+      {
+        type: mongoose.Schema.Types.ObjectId,
+        ref: "Message",
+      },
+    ],
+    memberId: [
```

Comment:
here why ref is from User when Member modal is present , take a look at design which i sent